stanford_image.strongarm.inc

```diff
@@ -175,6 +175,18 @@ function stanford_image_strongarm() {
       'stanford_huge_landscape' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_banner_850x400' => array(
+        'custom_settings' => TRUE,
+      ),
+      'gallery_large' => array(
```

Comment:
So these lines need to be removed?

      'gallery_large' => array(
        'custom_settings' => FALSE,
      ),
      'gallery_medium' => array(
        'custom_settings' => FALSE,
      ),
      'gallery_small' => array(
        'custom_settings' => FALSE,
      ),